packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -65,7 +65,8 @@ export class TdsDropdown {
   /** Normalizes input text for fuzzier search */
   @Prop() normalizeText: boolean = true;
 
-  /** Text that is displayed if filter is used and there are no options that matches the search. */
+  /** Text that is displayed if filter is used and there are no options that matches the search.
+   * Setting it to an empty string disables message from showing up. */
   @Prop() noResultText: string = 'No result';
```

Comment:
```suggestion
  @Prop() noResultText?: string;
```

Might be a better change to make this optional and undefined if it's not set. And then in your new check on line 573 would instead be:

```
{this.filterResult === 0 && this.noResultText  && (
```

Would be breaking though.

Reply:
Yep, dropping the message text and keeping it empty would be a breaking change as I think some of the users would get confused. However, I will add this to Tegel 2.0 to reconsider how we control showing the "No results" message.

Reply:
Great, should be made optional no though I think?

Reply:
Just added "?" to it ;)